pkg/operator/logutil/log.go

```diff
@@ -30,23 +30,15 @@ type goKitLogger struct {
 	l    log.Logger
 }
 
-// namedLogger gets log.Logger with component applied.
-func (l *goKitLogger) namedLogger() log.Logger {
-	logger := l.l
-	if l.name != "" {
-		logger = log.With(logger, "component", l.name)
-	}
-	logger = log.With(logger, l.kvps...)
-	return logger
-}
+var _ logr.LogSink = (*goKitLogger)(nil)
 
-func (l *goKitLogger) Log(keyvals ...interface{}) error {
-	return l.namedLogger().Log(keyvals...)
+func (l *goKitLogger) Init(info logr.RuntimeInfo) {
+	// no-op
 }
 
-func (l *goKitLogger) Enabled() bool { return true }
+func (l *goKitLogger) Enabled(level int) bool { return true }
```

Comment:
Is the intent to return if the specified log level is enabled? Or conversely are we hard coding to true to keep it the same as before?

Reply:
I'm just keeping the same behavior (where the logger is always enabled) but the interface changed so we had to add the unused parameter.